packages/e2e/docker-compose.yml

```diff
@@ -25,6 +25,7 @@ services:
       - ./local-network/network-files/node-sp1/:/root/network-files/node-sp1
       - ./local-network/config:/root/config
       - sdk-ipc:/sdk-ipc
+      - ./local-network/scripts:/root/scripts
```

Comment:
Wasn't it working without this change before? :thinking:

Reply:
Yes, this seems to be redundant

Reply:
I'll remove this.

Reply:
Removed on f755881